.github/ISSUE_TEMPLATE/tfaga-issue.yml

```diff
@@ -0,0 +1,43 @@
+name: Report an issue
+description: File a generic issue here
+body:
+  - type: markdown
+    attributes:
+      value: >
+        Thank you for taking the time to report this issue to us. We appreciate the
```

Comment:
Some nitpicking:

```
Thank you for taking the time to report this issue to us. We appreciate the feedback and use reports such as this one to guide our efforts in improving the `testing-farm-as-github-action`. That being said, our time resources are limited, and we cannot guarantee the timeliness or suitability of a resolution.

Feature requests are being evaluated separately. Contribution is highly appreciated.

For information on how to contribute to this GitHub repository, please visit the [Development Guide](https://github.com/sclorg/testing-farm-as-github-action/blob/main/DEVELOPMENT.md).
```

Reply:
Addressed by [2432e4b](https://github.com/sclorg/testing-farm-as-github-action/pull/183/commits/2432e4bdf7ae872b9a34671e3c5972b66ba6f5d6).

Reply:
I would prefer squashing all commits before merging to main.